### PR TITLE
rounded campaign icons to match every other icon on the website; clos…

### PR DESCRIPTION
**Pull request by Dylan-RG-Patterson**
…es #976
image

### CODE REVIEW

File: src/quest_manager/templates/quest_manager/category_list.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
   {% for object in object_list %}
   <tr>
     <td>{{ object.title }}</td>
-    <td>{% if object.icon %}<img src="{{ object.icon.url }}" width="50" height="50">{% endif %}</td>
+    <td>{% if object.icon %}<img class="img-responsive panel-title-img img-rounded" src="{{ object.icon.url }}" width="50" height="50">{% endif %}</td>
```
Comment (tylerecouture) [inline src/quest_manager/templates/quest_manager/category_list.html:24]: What does "panel-title-img" do here?  I think that css sets the image size, but we've already hard coded the image size on this line.... hmm. 

I think we can use the panel-title-img class and remove the width and height attributes?  We should be using css anyway and not hardcoding.

**Review by tylerecouture — CHANGES_REQUESTED** [review]
See inline comments

**Comment by tylerecouture** [comment]
@Dylan-RG-Patterson there's no test, which is fine since none of the other initialization stuff is tested... but can you verify that you've manually tested it by creating a new tenant and checking these new defaults show up?  Thanks!

**Comment by Dylan-RG-Patterson** [comment]
hopefully this is what was in mind.

**Pull request merged**